beta-pipelines/skylab/slidetags/SlideTags.wdl

```diff
@@ -0,0 +1,36 @@
+version 1.0
+
+import "scripts/spatial-count.wdl" as SpatialCount
+
+workflow SlideTags {
```

Comment:
This is super nit picky, but are we going with slidetags instead of slidetag? I'm good with whatever, just want to confirm